src/ProductConstructionService/ProductConstructionService.Cli/Operations/DeploymentOperation.cs

```diff
@@ -124,6 +115,17 @@ public async Task<int> RunAsync()
             _logger.LogInformation("Starting the service again");
             await StartActiveRevision();
         }
+
+        if (!string.IsNullOrEmpty(activeRevisionTrafficWeight.RevisionName))
```

Comment:
Not sure there is anything we want to do if the stopping of the old revision fails? At this point the new revision is activate and receiving traffic, don't think there is point in rolling back so I just let it error out here

Reply:
A warning would be nice? So that the pipeline goes orange?

Reply:
The exception is not handled, so the invocation would fail and the pipeline would go red, no?

Reply:
So we should handle it and emit a warning instead?